arch/arm/boot/dts/qcom/qcom-msm8974-samsung-hlte.dts

```diff
@@ -208,6 +233,14 @@
 		qcom,drive-strength = <PMIC_GPIO_STRENGTH_HIGH>;
 	};
 
+	touchkey_pin: touchkey-int-state {
+		pins = "gpio29";
+		function = "normal";
+		bias-disable;
+		input-enable;
```

Comment:
I think for 'tlmm' `input-enable` was removed across the board upstream, not sure what's the case for pm8941_gpios... Probably okay to keep since it wasn't removed from other nodes here

Reply:
For TLMM yes, `input-enable` is not a property, but for PMIC GPIOs it is valid. (I think)